operator/src/test/java/org/keycloak/operator/RealmImportE2EIT.java

```diff
@@ -25,6 +26,10 @@ public class RealmImportE2EIT extends ClusterOperatorTest {
     @Test
     public void testWorkingRealmImport() {
         // Arrange
+        // Recreating the database and the realm import CR to keep this test isolated
+        k8sclient.load(getClass().getResourceAsStream("/example-realm.yaml")).inNamespace(namespace).delete();
+        deleteDB();
+        deployDB();
```

Comment:
It seems we should isolate each test here. Maybe we could move this to a `@BeforeEach` method?

Reply:
not all, but most, ok I can refactor 👍